### DISCUSSION TITLE
Disable Explorer Shell security concerns

### DISCUSSION THREAD

**Discussion by fkm97** [SafeExamBrowser/seb-win-refactoring#733] [Q&A]
Hi,

I just have a question regarding security vulnerabilities. So, the scenario is as of right now SEB config files use "create new desktop" to conduct exams. Essentially, we use "Read&Write" for Speech-to-text. However, not everyone has it licensed. I did find a way to use "Voice Access" On windows 11 and it would do the same thing (and it is free). However, I have to change the kiosk mode to "Disable Explorer Shell" and start Voice Access before launching the SEB file. My question is what would be the security risks of using this instead of create new desktop?

**Comment by jbutt9** [comment]
Have you tested this within an exam? I can see the voice access ribbon at the top inside "Disable Explorer Shell" but can't get it to insert text anywhere. If you have a sanitized .seb file with your settings I'd love to see it.

**Reply by fkm97** [reply]
![20231130_120300](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/72949739/72062d76-f146-46ce-b81d-12371b3a9ecc)
[Test.zip](https://github.com/SafeExamBrowser/seb-win-refactoring/files/13515654/Test.zip)

This is a test file. If you launch voice access beforehand and launch it in "Disable Explorer Shell" ensuring that the cursor is already on the search tab. It works. I would just like to know the security risks of this method compared to the "Create New Desktop" method

**Reply by jbutt9** [reply]
Sorry to bug you on this again. Thanks for this test file, it turns out my machine was the only one having issues with VoiceAccess starting in Disable Explorer Shell mode. 

Have you figured out a way around VoiceAccess crashing when the banner at the top is clicked on? That's all I'm missing now for our students who need accomodations.

Thanks,

**Comment by dbuechel** [comment]
As long as you do have proper organizational security in conjunction with the usage of SEB (invigilation, random integrity checks of SEB installations etc.), I would say that it is perfectly fine to use kiosk mode Disable Explorer Shell. You'd just need to make sure that students are not able to cheat via the system accessibility functionalities.
